db/sg_replicate_cfg.go

```diff
@@ -1414,11 +1427,13 @@ func (l *ReplicationHeartbeatListener) reloadNodes() error {
 		}
 	}
 
+	localNodePresent = base.ContainsString(nodeUUIDs, l.mgr.localNodeUUID)
```

Comment:
Could avoid iterating twice here with `localNodePresent = true` above when looping through nodeSet to build up the nodeUUIDs list.